.gitignore

```diff
@@ -83,3 +83,5 @@ venv
 # vimmy stuff
 *.swp
 *.swo
+
+ctags.tmp
```

Comment:
This change just gave you merge conflicts because I already merged your other PR 👍